### DISCUSSION TITLE
New permission Access browsing history

### DISCUSSION THREAD

**Discussion by jarikmarwede** [floccusaddon/floccus#1638] [Q&A] [answered]
Hey, while updating Add-ons I got a prompt for granting a new permission. Is there an explanation anywhere about why the `Access browsing history` permission is now required? It doesn't seem obvious to me from the changelog alone, and I haven't found another discussion or issue about it.

**Comment by marcelklehr** [comment, marked as answer]
Hey @jarikmarwede 
The new permission is used for the new link statistics feature which sends bookmark usage statistics to your Nextcloud Bookmarks app, if you're using that sync method (it's opt-in of course). Otherwise the history permission is not used for anything else.

See [v5.2.0](https://github.com/floccusaddon/floccus/releases/tag/v5.2.0)

**Reply by aisknab** [reply]
Ok thank you. Just remember, the "alphabets" are on a full on frontal assault on community run browser extensions and other weak entry points as zero days become too expensive to purchase. Open source is easy to socially engineer in order to get as much data about people as possible.
